ansible/templates/usb/zones/esdc-cfgdb.vmmanifest.j2

```diff
@@ -26,6 +26,7 @@
   "customer_metadata": {
     "org.erigones:cfgdb_password": "@ESDC_INSTALL_PASSWORD@",
     "org.erigones:zabbix_ip": "@MON_IP@",
+    "org.erigones:erigonesd_ssl_cert": "@ERIGONESD_SSL_CERT@",
```

Comment:
The metadata name will be different. Please see #108.
I suggest that we unify the metadata in a way that both mgmt and cfgdb have a separate mdata for the key and cert:

* mgmt:
   * org.erigones:erigonesd_ssl_cert
   * org.erigones:erigonesd_ssl_key

* cfgdb (#108)
   * org.erigones:cfgdb_zk_rest_ssl_cer
   * rg.erigones:cfgdb_zk_rest_ssl_key